Golden_Path.postman_collection.json

```diff
@@ -12589,7 +12577,7 @@
 									"            });",
 									"            ",
 									"            pm.test(\"transactionId matches\", function () {",
-									"                pm.expect(data['transactionRequestId']).to.eql(pm.environment.get(\"transactionRequestId\"));",
+									"                pm.expect(data['transactionId']).to.eql(pm.environment.get(\"transactionRequestId\"));",
 									"            });",
```

Comment:
it is transactionRequestId in fact

Reply:
@elnyry For the callback endpoint, it is `transactionId`. I confirmed with the Swagger.

Reply:
correct